### DISCUSSION TITLE
Flex component & Space component

### DISCUSSION THREAD

**Discussion by nosferatu500** [ant-design/ant-design#45255] [General]
Hi!

I just updated to the latest version of antd (5.10) and noticed the new Flex component. I just checked this on the website and I'm not sure I quite understand the difference between the new Flex component and the Space component.

Can anyone describe what the difference is and when is it better to use Flex component vs Space component?

Thank you in advance!

**Comment by Wxh16144** [comment]
see more：https://github.com/ant-design/ant-design/discussions/44070
